prometheus_django/metrics.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import os
+from django.http import HttpResponse
+from prometheus_client import multiprocess
+from prometheus_client import generate_latest, CollectorRegistry, CONTENT_TYPE_LATEST
+
+
+# Create your views here.
+
+def metrics(request):
```

Comment:
Why do we have this in a diff repo and not in the django template repo? Would reduce the change of someone forgetting the repo in a new MS. 

This would make sense if we expect not to use prometheus in all MSes. Even then using a flag to enable/disable it may have been less error prone.

Reply:
@zahir-koradia  This repo is installed in all of the MS, adding it here would make life easier to change and maintain the code. DRY principle, you cannot use prometheus without this repo, so if you'd be using prometheus, you won't miss this.

Not sure if this makes sense.

Reply:
@zahir-koradia @rephus see usage here https://github.com/networklocum/application-service/pull/173/files

Reply:
Yeah agree with the idea of placing this here vs individually in each MS. But my thought was that if we expect to use prometheus in most/all MSes with django then we could have just used django-microservice template: https://github.com/networklocum/django-microservice-template and put everything in there. That would ensure this additional repo doesn't need to be include either in the template or in the repo.

Not a strong issue with the current setup. Just thought it would reduce the number of repos one needs to think about when working with MSes

Reply:
@zahir-koradia Yeah, gotcha, I can add that to django-microservice-template as well - maybe after we have moved all of them to gunicorn properly?

Reply:
Sounds good.